code/datums/status_effects/debuffs.dm

```diff
@@ -287,6 +288,78 @@
 	owner.underlays -= marked_underlay //if this is being called, we should have an owner at this point.
 	..()
 
+/datum/status_effect/eldritch
+	duration = 150 //15 seconds
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = null
+	var/mutable_appearance/marked_underlay
+	var/effect_sprite = ""
+
+/datum/status_effect/eldritch/on_apply()
+	if(owner.mob_size >= MOB_SIZE_HUMAN)
+		marked_underlay = mutable_appearance('icons/effects/effects.dmi', effect_sprite)
+		marked_underlay.pixel_x = -owner.pixel_x
+		marked_underlay.pixel_y = -owner.pixel_y
+		owner.underlays += marked_underlay
+		return TRUE
+	return FALSE
+
+/datum/status_effect/eldritch/proc/on_effect()
+	Destroy() //what happens when this is procced.
+
+/datum/status_effect/eldritch/Destroy()
+	if(owner)
+		owner.underlays -= marked_underlay
+	QDEL_NULL(marked_underlay)
+	return ..()
+
+/datum/status_effect/eldritch/be_replaced()
+	owner.underlays -= marked_underlay //if this is being called, we should have an owner at this point.
+	..()
+
+//Each mark has diffrent effects when it is destroyed that combine with the mansus grasp effect.
+/datum/status_effect/eldritch/flesh
+	id = "flesh_mark"
+	effect_sprite = "emark1"
+
+/datum/status_effect/eldritch/flesh/on_effect()
+	if(istype(owner,/mob/living/carbon/human))
+		var/mob/living/carbon/human/H = owner
+		H.bleed_rate += 2
+	..()
+
+/datum/status_effect/eldritch/ash
+	id = "ash_mark"
+	effect_sprite = "emark2"
+	var/repetitions = 1
+
+/datum/status_effect/eldritch/ash/on_creation(mob/living/new_owner, _repetition = 1)
+	repetitions = _repetition
+	. = ..()
+
+/datum/status_effect/eldritch/ash/on_effect()
+	if(istype(owner,/mob/living/carbon))
+		var/mob/living/carbon/C = owner
+		C.adjustStaminaLoss(10 * repetitions)
+		C.adjustFireLoss(5 * repetitions)
+		for(var/mob/living/carbon/C1 in range(1,C))
+			if((C1.mind && C1.mind.has_antag_datum(/datum/antagonist/ecult)) || C1 == C)
+				continue
+			C1.apply_status_effect(type,repetitions+1)
+			break
+	..()
+/datum/status_effect/eldritch/rust
+	id = "rust_mark"
+	effect_sprite = "emark3"
+
+/datum/status_effect/eldritch/rust/on_effect()
+	for(var/X in owner.GetAllContents())
+		if(istype(X,/obj/item))
+			var/obj/item/I = X
+			if(prob(75))
+				I.take_damage(rand(0,200))
```

Comment:
might wanna blacklist jumpsuits (and antimagic items) from this, because giving an ability an RNG chance to absolutely fuck someone over by destroying their jumpsuit mid-combat is gonna cause a lot of salt

Reply:
iirc it doesnt outright destroy them, but tears them apart, removing their armor values

Reply:
but yeah i will add a blacklist to it for anti-magic items.

Reply:
actually, nevermind, if they have an antimagic item on them, this will fail either way, since mansus grasp does an anti-magic check.